controller/officials.controller.go

```diff
@@ -175,3 +175,44 @@ func (h Handler) GetFormsByOfficialID(c *gin.Context) {
 		"data":     forms,
 	})
 }
+
+func (h Handler) VerifyUser(c *gin.Context) {
+	// Check the role of the user who sent the request
+	role := c.GetHeader("Role")
```

Comment:
whats with header  ? how does that works ?

Reply:
The GetHeader() function is used to retrieve the value of a specific header from an HTTP request , hear the specific header is Role  . When a client sends an HTTP request to a server, it includes various headers in the request. Headers contain additional information about the request or the client itself. Examples of common headers include "User-Agent" , "Content-Type" and  "Authorization"

Reply:
ya i know what a header is , this is not how it should be done